cmd/aad-cli/cli/user_test.go

```diff
@@ -174,6 +175,7 @@ func TestUserMoveHomeDirectory(t *testing.T) {
 		"fail if previous directory is absent": {prevHomeDir: "absent", newHomeDir: "newhome", wantErr: true},
 		"fail if previous directory is a file": {prevHomeDir: "oldhomefile", newHomeDir: "newhome", wantErr: true},
 		"fail if new directory already exists": {prevHomeDir: "oldhome", newHomeDir: "existingnewhome", wantErr: true},
+		"fail if the user has open processses": {prevHomeDir: "oldhome", newHomeDir: "newhome", userLoggedIn: true, wantErr: true},
```

Comment:
Nitpick (just to have one): I would add something here too that we keep the old attribute. (hence the userLoggedIn: true, but I find it a little bit puzzling)